kiwi/volume_manager/btrfs.py

```diff
@@ -267,6 +270,9 @@ def sync_data(self, exclude=None):
             sync_target = self.mountpoint + '/@'
             if self.custom_args['root_is_snapshot']:
                 sync_target = self.mountpoint + '/@/.snapshots/1/snapshot'
+                self._create_snapshot_info(
+                    self.mountpoint + '/@/.snapshots/1/info.xml'
+                )
```

Comment:
Guys-guys. Please remember [this](http://kotsf.com/gdarchive/7/), [this](https://wiki.python.org/moin/PythonSpeed/PerformanceTips) and [this](http://www.wikihow.com/Avoid-Repeating-the-Old-Same-Mistakes-All-over-Again).